pkg/util/version/stream.go

```diff
@@ -4,8 +4,8 @@ package version
 // Licensed under the Apache License 2.0.
 
 type Stream struct {
-	Version  *Version
-	PullSpec string
+	Version  *Version `json:"version"`
+	PullSpec string   `json:"pullSpec"`
```

Comment:
Do we really want pull spec in this json? Or should we ommit?

Reply:
I was thinking of omitting it. You've just convinced me :)

Reply:
@m1kola Changed

Reply:
@nilsanderselde sorry, my comment was ambiguous: I ment not marshaling pull spec at all (always omit, ``json:"-"``).

I beleive `omitempty` doens't make any difference becuase we always have both values in our codebase, if I follow correctly.

Reply:
I had defined `MarshalJSON` to make a `type stream Stream` and set `PullSpec` to `""`, so `omitempty` did work to exclude it.

However, I didn't know `json:"-"` was an option — that's much simpler. Changed.